### DISCUSSION TITLE
Product Select Options Not Showing as Selected on Front End

### DISCUSSION THREAD

**Discussion by Childzy** [swellstores/community#156] [Help] [answered]
Hi,

New to Swell and GitHub - I have a product with Male/Female options but on iPhone when selecting the option, there is no visual reference being made to show it is selected.

Assuming this is a bug - anyone aware of a workaround?

https://www.just-check.co.uk/products/choose-your-plan/

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi @Childzy, it does looks like a small UI bug when using mobile browsers. We'll add this fix to our backlog, thank you for the heads up.

I'll let you know when it gets updated here.

**Reply by Childzy** [reply]
Thanks for the quick response - we are launching our beta on Monday 6th. Assuming fix wont be quite that quick!

Is there any workaround I can deploy my side? Custom CCS etc?

**Reply by rodrigoddalmeida** [reply]
There is a temporary workaround you can use in the meantime. In `Theme Editor > Design & global settings > Global scripts` you should be able to add content to the `<head>` tag. Here's an example of how that content might look like:

```html
<script>
  /* Empty <script> tag, it forces the theme to allow this content to be added raw to the <head> tag. */
</script>
<style type="text/css">
  /* Add your custom CSS here. */
</style>
```

You might need to refresh the editor review frame to see the custom CSS in action.
